city_scrapers/spiders/il_elections.py

```diff
@@ -0,0 +1,148 @@
+from datetime import datetime
+
+import scrapy
+from city_scrapers_core.constants import BOARD
+from city_scrapers_core.items import Meeting
+from city_scrapers_core.spiders import CityScrapersSpider
+
+
+class IlElectionsSpider(CityScrapersSpider):
+    name = "il_elections"
+    agency = "Illinois State Board of Elections"
+    timezone = "America/Chicago"
+    start_urls = ["https://www.elections.il.gov/AboutTheBoard/MeetingMinutesAll.aspx"]
+    custom_settings = {"ROBOTSTXT_OBEY": False}
+
+    def __init__(self):
+        self.meeting_minutes = dict()
+        self.addresses = dict()
+        super().__init__()
+
+    def parse(self, response):
+        """
+        `parse` should always `yield` Meeting items.
+
+        Change the `_parse_title`, `_parse_start`, etc methods to fit your scraping
+        needs.
+        """
+        # parse meeting minutes and addresses first to associate them with the agenda
+        self._parse_minutes(response)
+        self._parse_addresses(response)
+
+        # parse the agenda
+        yield scrapy.Request(
+            "https://www.elections.il.gov/AboutTheBoard/Agenda.aspx",
+            callback=self._parse_agenda,
+            dont_filter=True,
+        )
+
+    def _parse_minutes(self, response):
+        """Parse all past board meeting minutes, store for association to meetings."""
+        minutes = response.css("#ContentPlaceHolder1_gvMeetingMinutes td > a")
+        for minute in minutes:
+            date = minute.css("::text").extract_first().strip()
+            parsed_date = datetime.strptime(date, "%a, %B %d, %Y")
+            link = minute.css("::attr(href)").extract_first()
```

Comment:
It looks like the `href` attribute opens up to a viewer page. We want to parse out everything that comes after the query parameter starting with `/Downloads` so that we can get the PDF link directly